src/common/utxobased/db/newProcessor.ts

```diff
@@ -151,6 +157,16 @@ export async function makeNewProcessor(
       baselets = await makeBaselets({ disklet })
     },
 
+    async dumpData(): Promise<DumpDataReturn[]> {
+      const allBases = Object.values(baselets.all)
+      return await Promise.all(
+        allBases.map(async base => ({
+          databaseName: base.databaseName,
+          data: (await base.dumpData('')) as never
+        }))
+      )
+    },
```

Comment:
What is the reason for `dumpData()` method? I'm assuming it'll be needed in the engine, but for what or where?